spikewidgets/widgets/multicompgraphwidget/multicompgraphwidget.py

```diff
@@ -221,14 +235,20 @@ def _do_plot(self):
         sg_names, sg_units = self._msc.compute_subgraphs()
         # fraction of units with agreement > threshold
         for i, name in enumerate(name_list):
-            ax = self.get_tiled_ax(i, ncols=len(name_list), nrows=1)
+            if self._ax is not None:
+                if len(self._ax)>1:
```

Comment:
I think that the best way to handle this is to set the `self._axes` and modify the `get_tiled_ax` to return one of the axes (if present) or create new ones if not